packages/mobile/src/account/AccountKeyEducation.tsx

```diff
@@ -41,30 +47,10 @@ function useSteps() {
   return React.useMemo(
     () =>
       [
-        {
-          image: null,
-          cancelEvent: AnalyticsEvents.backup_educate_1_cancel,
-          progressEvent: AnalyticsEvents.backup_educate_1_next,
-          screenName: 'AccountKeyEducation',
-        },
-        {
-          image: null,
-          cancelEvent: AnalyticsEvents.backup_educate_2_cancel,
-          progressEvent: AnalyticsEvents.backup_educate_2_next,
-          screenName: 'AccountKeyEducation',
-        },
-        {
-          image: null,
-          cancelEvent: AnalyticsEvents.backup_educate_3_cancel,
-          progressEvent: AnalyticsEvents.backup_educate_3_next,
-          screenName: 'AccountKeyEducation',
-        },
-        {
-          image: null,
-          cancelEvent: AnalyticsEvents.backup_educate_4_cancel,
-          progressEvent: AnalyticsEvents.backup_educate_4_next,
-          screenName: 'AccountKeyEducation',
-        },
+        { image: null, topic: EducationTopic.backup },
+        { image: null, topic: EducationTopic.backup },
+        { image: null, topic: EducationTopic.backup },
+        { image: null, topic: EducationTopic.backup },
```

Comment:
What is this change meant to do?

Reply:
This is meant to simplify and reduce the data we are passing through to the `Education` screen. All we need to know is where the data is coming from and in the `Education` methods themselves we have the logic for determining which event to log.

I think this helps to keep analytics logic a little more contained.